src/commonControls/multiValueListControl/MultiValueListControl.ts

```diff
@@ -949,15 +993,39 @@ export class MultiValueListControl extends Control implements InteractionModelCo
     private handleRemoveChoiceByTouch(input: ControlInput, resultBuilder: ControlResultBuilder) {
         const choiceIndex = (input.request as interfaces.alexa.presentation.apl.UserEvent).arguments![2];
         assert(choiceIndex !== undefined);
+        const touchAction = (input.request as interfaces.alexa.presentation.apl.UserEvent)
+            .arguments![1] as string;
         const content = this.getSlotIds();
-        const choiceId = content[choiceIndex - 1];
-        this.state.value?.splice(choiceIndex - 1, 1);
-        resultBuilder.addAct(
-            new ValueRemovedAct(this, {
-                value: choiceId,
-                renderedValue: this.evaluateRenderedValue(choiceId, input),
-            }),
-        );
+        if (touchAction === 'Remove') {
+            const choiceId = content[choiceIndex - 1];
```

Comment:
refactor: move to separate function this.removeItem()?

Reply:
updated